[BUG] Error building hdk_derive

**Describe the bug**
I just scaffolded a new repo, bumped everything to 0.2.2 and was able to build as expected. 

Then I tried to add sweettest with the following:
```
[dev-dependencies]
holochain = { version = "0.2.2", default-features=false, features=["test_utils"]}
```

Now its failing to build `hdk_derive`. I get the following error:

```
[holonix:~/Projects/Holochain/bug-reproduction-validation-multi-zome]$ cargo check
+ command cargo check
    Blocking waiting for file lock on build directory
   Compiling hdk_derive v0.2.2
error: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/nix/store/bd51ij6c70cf6za578wraf17x8h21n93-rust-default-1.71.1/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/.cargo/bin:/nix/store/18bs92p6yf6w2wwxhbplgx02y6anq092-gcc-wrapper-12.3.0/bin:/nix/store/h5kvfrjmpw792v8jg7nrzfkffmn0iyy8-gcc-12.3.0/bin:/nix/store/f6in5kb2y5v06zinz1a6xy6cyg67q026-glibc-2.37-8-bin/bin:/nix/store/y9gr7abwxvzcpg5g73vhnx1fpssr5frr-coreutils-9.3/bin:/nix/store/mc6q3cdz5s0p1aj4y586bglsfsnsf2k8-binutils-wrapper-2.40/bin:/nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin:/nix/store/x8vrc1c6r8nn454whr99bxc5hdy5ybjf-nodejs-18.18.0/bin:/nix/store/4mkpgb01fhp5yl5w9v70f882lgp0darn-holochain-workspace/bin:/nix/store/361fg7q8n5q2dr3i1h8nd5i930yl8w3a-lair-keystore-0.0.1/bin:/nix/store/fr3mq69c03s38l6f3inclp131w0wzh7w-hc-launch-0.0.1/bin:/nix/store/brc11kw0sb0i46mbvznfpih3vjhah0r4-hc-scaffold-0.1.11/bin:/nix/store/8apjgyfrz5qi0y5ihrs2dgajvm08qz4g-hn-introspect/bin:/nix/store/2j7b1ngdvqd0bidb6bn9icskwm6sq63v-perl-5.38.0/bin:/nix/store/5daca24rn22c65ff25lc6z0g0imfphvr-pkg-config-wrapper-0.29.2/bin:/nix/store/23wrmhyfnwjxc9cvy3lab9xj551ic53c-go-1.20.8/bin:/nix/store/3lwrcxnxvfskh7m98fq0n5kqj2001ci7-jq-1.7-bin/bin:/nix/store/bd51ij6c70cf6za578wraf17x8h21n93-rust-default-1.71.1/bin:/nix/store/33vlrbngipikrfyhgyrpjars5qrjacj2-zstd-1.5.5-bin/bin:/nix/store/5p7jmbpmq7clb3whbi6kyp50gmg54l2d-zstd-1.5.5/bin:/nix/store/ay0p9mbw1w3zkvwzx3c94xq7x8jrn9wq-patchelf-0.15.0/bin:/nix/store/8i08qh6yrxl8cw31ari0ljx78chfk6qd-openssl-3.0.10-bin/bin:/nix/store/m74r97lyqflmacyayli8y5p1aagsnb94-openssl-static-x86_64-unknown-linux-musl-3.0.10-bin/bin:/nix/store/h6ddd0s2m11rzmf9nz85zgf8vy76qnvg-sqlcipher-4.5.5/bin:/nix/store/y9gr7abwxvzcpg5g73vhnx1fpssr5frr-coreutils-9.3/bin:/nix/store/b6izr8wh0p7dyvh3cyg14wq2rn8d31ik-findutils-4.9.0/bin:/nix/store/q56n7lhjw724i7b33qaqra61p7m7c0cd-diffutils-3.10/bin:/nix/store/x23by79p38ll0js1alifmf3y56vqfs49-gnused-4.9/bin:/nix/store/xafzciap7acqhfx84dvqkp18bg4lrai3-gnugrep-3.11/bin:/nix/store/8kkn44iwdbgqkrj661nr4cjcpmrqqmx8-gawk-5.2.2/bin:/nix/store/89s3w7b4g78989kpzc7sy4phv0nqfira-gnutar-1.35/bin:/nix/store/2a9na7bp4r3290yqqzg503325dwglxyq-gzip-1.13/bin:/nix/store/gxknjk51s7q86llkbzpaqv43kflj9d8j-bzip2-1.0.8-bin/bin:/nix/store/2jp6cv2q4cgh91f5lp57p945rq98ldhr-gnumake-4.4.1/bin:/nix/store/xdqlrixlspkks50m9b0mpvag65m3pf2w-bash-5.2-p15/bin:/nix/store/c15ama0p8jr4mn0943yjk4rpa2hxk7ml-patch-2.7.6/bin:/nix/store/sb3sxnp4g40gfw758a0m4sjm7slvmax9-xz-5.4.4-bin/bin:/nix/store/xfjqspcc9442hi0lm0szv3sw75zswvml-file-5.45/bin:/home/matt/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/home/matt/.asdf/shims:/home/matt/.asdf/bin:/home/matt/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/matt/.local/share/JetBrains/Toolbox/scripts:/opt/android-studio/bin:/opt/android-sdk/platform-tools/:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/lib:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/git:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/heroku:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/pip:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/lein:/home/matt/.antigen/bundles/robbyrussell/oh-my-zsh/plugins/command-not-found:/home/matt/.antigen/bundles/zsh-users/zsh-syntax-highlighting" VSLANG="1033" "cc" "-Wl,--version-script=/tmp/nix-shell.f8r8q4/rustcSGNGQs/list" "-Wl,--no-undefined-version" "-m64" "/tmp/nix-shell.f8r8q4/rustcSGNGQs/symbols.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.0.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.1.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.10.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.11.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.12.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.13.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.14.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.15.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.2.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.3.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.4.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.5.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.6.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.7.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.8.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.hdk_derive.e632b739a4ca0efe-cgu.9.rcgu.o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.3xtfk98gik4wxs2o.rcgu.rmeta" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/hdk_derive-d89d9036e18cf1f6.59b328miy7vy34cf.rcgu.o" "-Wl,--as-needed" "-L" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps" "-L" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/build/libsodium-sys-stable-1f5bf0854f998373/out/installed/lib" "-L" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/build/libsqlite3-sys-e54cf88dc7b9fa97/out" "-L" "/nix/store/bd51ij6c70cf6za578wraf17x8h21n93-rust-default-1.71.1/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libheck-1658eeab85cd3e25.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libdarling-93685d7cbffb9b97.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libdarling_core-9068d9dd78f3579d.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libstrsim-fe0f5161373ed15a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfnv-3125ae6b7241603c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libident_case-f78df3bd73e5d3b6.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libholochain_integrity_types-b9041df1cb104a4f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libkitsune_p2p_dht-88a1c40e7eec6b97.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libstatrs-a6a0f5687f65deb5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnalgebra-92b33f4064c0baf0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librand_distr-5affbfb68e9cfd70.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_rational-ca82719545118b95.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_bigint-a2ebdb72d6e6440f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtypenum-b5e32b2161941b9d.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsimba-436babc36940f2db.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_complex-692e962822a88148.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmatrixmultiply-b6a00366b72f3de8.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librawpointer-0e907f60bd3cc5a5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libapprox-ab0f8a06b8279b0a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsubtle_encoding-70fe492494fbeac3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libzeroize-58bbc75d46ffa48b.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsubtle-13eb10c4b5b664c3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libkitsune_p2p_timestamp-abbe23c81d34d070.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libchrono-c39e5d5505063b47.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libiana_time_zone-bce5afa326430654.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libholo_hash-7b3d9e0b5d84cf5f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libblake2b_simd-8b3eaa5a82b5306f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libconstant_time_eq-120850b078afc529.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libarrayvec-c22b78beb63aa6d6.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libarrayref-0dd604093878c688.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbase64-6015bbd7d137e88c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libholochain_util-14c91b9ff9a74cbb.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librpassword-98307cc4ed96675c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librtoolbox-b6b5af54d7c48ab5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsodoken-0201629245309b05.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblibsodium_sys-f6d5f8e07c9ae93a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libone_err-815d4f42a9bf3fe4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libdunce-9cfe54653721b460.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtokio-937cf38f77f396e0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsignal_hook_registry-7c111ddb561f5a04.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_cpus-6e7f951917f56581.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsocket2-ec3c1a98ac73dc16.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbytes-18172e161c1813d7.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmio-95cd2f9d224704b6.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libparking_lot-e48a2f8efe5c314c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libparking_lot_core-9d93f6fb5f11dd9a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblock_api-a09067cf2d01767c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfixt-0fdafe211e7e515e.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libparking_lot-41ca4e04bf4e9360.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libparking_lot_core-7dbf84e547bd6673.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcfg_if-2f1451f7ec97ada7.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblock_api-b5e94da6f580323c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libstrum-5cb2167c882c6d22.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librand-624c105c89822d72.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librand_chacha-10430b69fd7b13a8.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libppv_lite86-d1b5f94499a35c89.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librand_core-08bc20f8577e0b53.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmust_future-69c7a62f7dfed093.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures-a0574eceefa5e69d.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_executor-79749cfe31954d44.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_util-305a54c6add65ffb.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_io-8611a1fec5c4c6aa.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libslab-257db4129338a02b.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_channel-7a63aff411038e66.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_sink-d73a9972dba9e5bc.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_task-bd7cde4b6ebbe484.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libpin_utils-6407450b632eb0e0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfutures_core-2ee040b07a0e7045.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libkitsune_p2p_dht_arc-d0f02056ce254415.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librusqlite-8ac44c8e5dbad665.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfallible_streaming_iterator-77e0c46d6fc0e6b4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libhashlink-b6dba953ca02c0d3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblibsqlite3_sys-8bb1e7e7c6f610a1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libinterval-4c8a8d84865d3bdf.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libgcollections-fd2534e2f2104ec2.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtrilean-0afa9a2fda335f00.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbit_set-97ac873f52a4c6e2.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbit_vec-d74c3591b4053f47.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_integer-463ef856d794f57f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libholochain_wasmer_common-fd9efa80f242cbb8.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer-ab42ec1e23c79b46.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_engine_dylib-4ec396fca69b2aa1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwhich-ccfe8c839450c214.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libhome-c27eb2dbd9d641f4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_object-c0253de6311fe7e5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtempfile-465bedfa10b98588.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfastrand-1bbee9457d783476.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librustix-e82beacf10d9e3d7.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbitflags-787eb0ad26aecb56.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblinux_raw_sys-5ef94228e1e2d435.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libobject-97991fdc91103a53.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcrc32fast-0412d93dd96d4a53.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblibloading-ba707f4ddd412038.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_engine_universal-749de47494a6cd97.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_engine_universal_artifact-4564b73d114aef9d.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_compiler_cranelift-8b296afc084864b3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtracing-06290c50850f34dc.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libpin_project_lite-78225b4d3a204ffc.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtracing_core-1e737c27458b97cb.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcranelift_frontend-99e6b0328a95b3ef.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librayon-45b25e221a667bdd.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librayon_core-81033e86ba5f0644.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcrossbeam_deque-8b0a2e0a22dba6ba.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcrossbeam_epoch-40aaeb59b7dd473e.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmemoffset-791fcc5a49174d49.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcrossbeam_utils-dfbad8b150c996f9.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libeither-2ebec5ad2f215a20.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcranelift_codegen-caa80256e51b8244.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcranelift_codegen_shared-affb30ccb4fec5a4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libregalloc-45fc4f002c44fc5c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librustc_hash-2d2234f13b057222.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblog-8d1fe8415f611694.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libvalue_bag-ae748ac38d49f8d9.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libgimli-badb1c76d19e0e07.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libfallible_iterator-82403dca79f95c7c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libstable_deref_trait-776cf93089a109e4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcranelift_bforest-9ec88f78752c055c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcranelift_entity-cb7f9d3cf4306140.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwat-68484c26dcd792f6.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwast-87cb87404e2d4383.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasm_encoder-a3cc621b539ac91b.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libleb128-6becc645ed3a5cbf.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libunicode_width-14008df63c334f2e.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_engine-35cb98e20f2133cb.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmemmap2-e6d1214210c8485a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_vm-84320097454277ec.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libregion-b818683c34e5a581.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbitflags-7c3fea527d77b1c9.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblazy_static-8671b3500b847cba.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcorosensei-3d6b02ec5973f428.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libscopeguard-87cfc61d47527392.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbacktrace-c432c1a83dd0c051.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libminiz_oxide-ad645db778ac1dd4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libadler-041d8e6d418a9b74.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libobject-6a54dcc7c07cf696.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmemchr-70f768fa561f7f47.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libaddr2line-6df3352aa1208152.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libgimli-669f0cc95b42ab1e.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librustc_demangle-14aae7ca2d225788.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmemoffset-22e0b4853afeb90b.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_artifact-ba7bd2142990bff5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_compiler-e580f48d258d6a85.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsmallvec-807bcb1dd0c7581c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libtarget_lexicon-f249c8084b2c959e.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmparser-407cece4254c4ae1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libwasmer_types-4e7551d4e7efa1ce.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libmore_asserts-1920f043f77a4681.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libenum_iterator-080bdc9fdec41ad3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librkyv-44414c0aaee2804f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libseahash-36c5ec971a7068da.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libptr_meta-276e5c9a0446a1ce.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libenumset-05c343585010d9f8.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libloupe-e7d9f7f73a9b532a.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libindexmap-60d780bc4c8e31a9.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libhashbrown-bffc70d24265c609.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libahash-3f3a79ec6746247f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libholochain_serialized_bytes-f3d83531c3e3e7a5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libserde_transcode-a446a9058f949dd1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libserde_bytes-225adaed394e3fab.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libarbitrary-eef1caf9e8c7adf9.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libthiserror-ba71276dccfcca39.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librmp_serde-aa1e57bdfeec44b1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/librmp-5ffe15e5d864e0e5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libbyteorder-c9854a8ad35d4cc5.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libnum_traits-769a98a5848510fb.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libserde_json-6ef847f0d050d0a0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libryu-383452ca37b2504f.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libitoa-2d42945e012699c4.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libindexmap-4ef94c1deb8b853c.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libequivalent-ff87fc2bf7832150.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libhashbrown-99550c42fedfeaf0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libahash-39985874bcd77014.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libgetrandom-2f9968cc17de9405.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liblibc-37d72999b0121c43.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libonce_cell-9c36602133ffb192.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libcfg_if-981ad9e7e5270aa0.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/liballocator_api2-474639e1765700d6.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libserde-f08b2ad0b2ec9889.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libproc_macro_error-06385cd6a82d9def.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libsyn-d627fa924239e718.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libquote-ca150d7553b733f3.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libproc_macro2-5270222506e6ced1.rlib" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libunicode_ident-680c6e6419e83968.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libproc_macro-918a4c8e5082c14b.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-7b9f6349d87c69a1.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-8aa13c9d539a65d0.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-584d799a3f3eb3c4.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libmemchr-31c151ca16df12cb.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-c7816532343be4b4.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-b9b833ec51690ba4.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-77071c92726d4076.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-e299e57d9e5c1111.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-2c7771df0549e461.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-ea1a747db17b6836.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-0e3146805ce934d6.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-464690d32269d503.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-6563489380ff8725.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-eeca9ee8c0120aaf.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-d302b9c70aefff51.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-99d811a5f83caaf4.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-699fe6dfc31ade7e.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-4efaba4fe03302d5.rlib" "/nix/store/pq2q0d0c1fbpx665dcad1bylk9b1lg1g-rust-std-1.71.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-d092fa2c0bdfc89c.rlib" "-Wl,-Bdynamic" "-ldl" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/nix/store/bd51ij6c70cf6za578wraf17x8h21n93-rust-default-1.71.1/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/matt/Projects/Holochain/bug-reproduction-validation-multi-zome/target/debug/deps/libhdk_derive-d89d9036e18cf1f6.so" "-Wl,--gc-sections" "-shared" "-Wl,-z,relro,-z,now" "-nodefaultlibs"
  = note: /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_size: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_set: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_init: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_grow: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_get: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_fill: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_table_copy: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_raise_trap: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_probestack: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_memory32_size: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_memory32_init: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_memory32_grow: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_memory32_fill: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_memory32_copy: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_table_size: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_table_set: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_table_grow: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_table_get: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_memory32_size: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_memory32_grow: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_memory32_fill: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_imported_memory32_copy: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_func_ref: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f64_trunc: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f64_nearest: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f64_floor: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f64_ceil: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f32_trunc: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f32_nearest: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f32_floor: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_f32_ceil: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_externref_inc: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_externref_dec: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_elem_drop: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: wasmer_vm_data_drop: undefined version: 
          /nix/store/74y3751gsixaz9797ib0hp7c658sp1y5-binutils-2.40/bin/ld: failed to set dynamic section sizes: bad value
          collect2: error: ld returned 1 exit status
          

error: could not compile `hdk_derive` (lib) due to previous error

[holonix:~/Projects/Holochain/bug-reproduction-validation-multi-zome]$ 
```

**Expected behavior**
hdk_derive builds on latest 0_2 holonix

**System information:**
 - OS: Ubuntu 22.04
 - Browser: n/a
 - Holochain and HDK Version: 0.2.2
 
**Steps to reproduce**
Repo: https://github.com/mattyg/bug-reproduction-validation-multi-zome

--- a/nix/modules/devShells.nix
+++ b/nix/modules/devShells.nix
@@ -19,7 +19,7 @@
 
       devShells = {
         default = self'.devShells.holonix;
-        holonix = pkgs.mkShell {
+        holonix = pkgs.mkShell.override ({ stdenv = config.rustHelper.defaultStdenv pkgs; }) {
           inputsFrom = [ self'.devShells.rustDev ];
           packages = holonixPackages ++ [ hn-introspect ];
           shellHook = ''
@@ -92,7 +92,7 @@
                 );
 
           in
-          pkgs.mkShell {
+          pkgs.mkShell.override ({ stdenv = config.rustHelper.defaultStdenv pkgs; }) {
             inputsFrom = [ self'.devShells.rustDev ] ++ (
               # filter out the holochain binary crates from the shell because it's at best unnecessary in local development
               # it's currently a nativeBuildInput because one of the unit tests requires `holochain` and `hc-sandbox` in PATH
@@ -161,7 +161,7 @@
           };
 
         rustDev =
-          pkgs.mkShell
+          pkgs.mkShell.override ({ stdenv = config.rustHelper.defaultStdenv pkgs; })
             {
               inputsFrom = [
                 self'.packages.holochain

